GameData/KerbalismConfig/Support/ROCapsules.cfg

```
@@ -601,26 +1002,47 @@
 {
 	@MODULE[Habitat]
 	{
-		%volume = 8.95
-		%surface = 35
+		%volume = 9	//m^3
+		%surface = 4.3267	//m^2
+		%max_pressure = 0.35
 	}
 }
 @PART[ROC-CSTCM]:NEEDS[ROCapsules,FeatureHabitat]:AFTER[zzzKerbalism]
 {
 	@MODULE[Habitat]
 	{
-		%volume = 11
-		%surface = 35	//guess, very similar dimensions to Orion
+		%volume = 11	//m^3
+		%surface = 4.9461	//m^2
```

```suggestion
		%surface = 35	//m^2
```
Impossible for that internal volume.